force-app/main/default/classes/AccountModelSettingsVMapper_TEST.cls

```diff
@@ -138,10 +135,158 @@ public with sharing class AccountModelSettingsVMapper_TEST {
+     * correspond to all active Account Record Types in the org.
+     ***********************************************************************************************************************************/
+    @isTest
+    private static void getAccountModelSettingsViewModelValid() {
+        RecordTypeMapper.instance = new STUB_RecordTypeMapper();
+
+        Hierarchy_Settings__c hierarchySettings = UTIL_CustomSettings_API.getSettingsForTests(
+            new Hierarchy_Settings__c(
+                Account_Processor__c = '000000000000000001',
+                Household_Addresses_RecType__c = '',
+                Administrative_Account_Record_Type__c = null
+            )
+        );
+
+        AccountModelSettingsVMapper accountModelSettingsVMapperInstance = AccountModelSettingsVMapper.getInstance();
+
+        Test.startTest();
+        AccountModelSettingsVModel accountModelSettingsVModel = accountModelSettingsVMapperInstance.getAccountModelSettingsViewModel();
+        Test.stopTest();
+
+        System.assertEquals(
+            '000000000000000001',
+            accountModelSettingsVModel.defaultAccountRecordType.value,
+            'Selected value for default account record type should be set as default account.'
+        );
+
+        System.assertEquals(
+            '""',
+            accountModelSettingsVModel.householdAccountRecordType.value,
+            'Selected value for household account record type should be set as household account.'
+        );
+
+        System.assertEquals(
+            '""',
+            accountModelSettingsVModel.adminAccountRecordType.value,
+            'Selected value for admin account record type should be set as admin account.'
+        );
+
+        System.assertEquals(
+            1,
+            accountModelSettingsVModel.defaultAccountRecordType.options.size(),
+            'Should only list active record types as options.'
+        );
+        System.assertEquals(
+            '000000000000000001',
+            accountModelSettingsVModel.defaultAccountRecordType.options[0].value,
+            'Should only list active record types as options.'
+        );
+    }
+
+    /***********************************************************************************************************************************
+     * @description Test method to verify that the getAccountModelSettingsViewModel returns an AccountModelSettingsVModel with the selected
+     * value set to the value specified in the associated Hierarchy Settings fields and an empty list of comboboxVModels when there are
+     * no active Account Record Types in the org.
+     ***********************************************************************************************************************************/
+    @isTest
+    private static void getAccountModelSettingsViewModelWithNoRecordTypes() {
+        // TODO: what happens if there are zero Account record types in the org?
```

Comment:
Change this to EDA being wholly broken without record types in the org, therefore consideration of that scenario is unnecessary for unit tests.
```suggestion
        // TODO: what happens if there are zero Account record types in the org?
```